### PR TITLE
fix missing share icon

### LINKED ISSUE #34970: [Bug]: Talk icon missing when sharing a file to a conversation
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

When searching for a user to share a file, the talk icon is missing:
image

image

(Tested on NC 25.0.1)

### Steps to reproduce

1. Share a file
2. Search for a user which also has a talk conversation
3. Notice that user and conversation are not distinguishable

### Expected behavior

User and conversation should be distinguishable.

Changing to:
image

Works:
image

### Installation method

Community Manual installation with Archive

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 8.1

### Web server

Apache (supported)

### Database engine version

PostgreSQL

### Is this bug present after an update or on a fresh install?

Updated to a major version (ex. 22.2.3 to 23.0.1)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

_No response_

### List of activated Apps

```shell
Enabled:
  - activity: 2.17.0
  - admin_audit: 1.15.0
  - circles: 25.0.0
  - cloud_federation_api: 1.8.0
  - comments: 1.15.0
  - dashboard: 7.5.0
  - dav: 1.24.0
  - deck: 1.8.1
  - federatedfilesharing: 1.15.0
  - federation: 1.15.0
  - files: 1.20.1
  - files_pdfviewer: 2.6.0
  - files_rightclick: 1.4.0
  - files_sharing: 1.17.0
  - files_trashbin: 1.15.0
  - files_versions: 1.18.0
  - forms: 3.0.1
  - integration_github: 1.0.12
  - logreader: 2.10.0
  - lookup_server_connector: 1.13.0
  - notifications: 2.13.1
  - notify_push: 0.5.0
  - oauth2: 1.13.0
  - password_policy: 1.15.0
  - photos: 2.0.0
  - polls: 4.0.0
  - previewgenerator: 5.1.0
  - privacy: 1.9.0
  - provisioning_api: 1.15.0
  - related_resources: 1.0.3
  - richdocuments: 7.0.1
  - serverinfo: 1.15.0
  - settings: 1.7.0
  - sharebymail: 1.15.0
  - spreed: 15.0.1
  - systemtags: 1.15.0
  - tasks: 0.14.5
  - text: 3.6.0
  - theming: 2.0.1
  - twofactor_backupcodes: 1.14.0
  - updatenotification: 1.15.0
  - user_status: 1.5.0
  - viewer: 1.9.0
  - weather_status: 1.5.0
  - workflowengine: 2.7.0
Disabled:
  - bruteforcesettings: 2.4.0
  - calendar: 4.1.0
  - checksum: 1.1.5
  - collectives: 2.0.0
  - contacts: 5.0.1
  - contactsinteraction: 1.4.0
  - encryption
  - files_antivirus: 3.3.1
  - files_automatedtagging: 1.15.0
  - files_external
  - files_retention: 1.14.0
  - firstrunwizard: 2.12.0
  - flow_notifications: 1.4.0
  - groupfolders: 13.0.0
  - guests: 2.3.0
  - integration_whiteboard: 0.0.15
  - issuetemplate: 0.7.0
  - keeweb: 0.6.10
  - metadata: 0.17.0
  - nextcloud_announcements: 1.5.0
  - onlyoffice: 7.6.8
  - ransomware_protection: 1.14.0
  - recommendations: 0.4.0
  - support: 1.6.0
  - survey_client: 1.11.0
  - suspicious_login
  - theming_customcss: 1.12.0
  - twofactor_totp
  - user_ldap
  - workflow_pdf_converter: 1.10.0
  - workflow_script: 1.9.0
```

### Nextcloud Signing status

_No response_

### Nextcloud Logs

_No response_

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by szaimen**
Fix https://github.com/nextcloud/server/issues/34970

Signed-off-by: Simon L <szaimen@e.mail.de>

### CODE REVIEW

File: apps/files_sharing/css/icons.scss [hunk 1]
```diff
@@ -24,11 +24,11 @@
...
-	@include icon-color('app', 'spreed', variables.$color-black);
+	background-image: var(--icon-talk-dark);
 }
 .icon-circle {
-	@include icon-color('circles', 'circles', variables.$color-black, 3, false);
+	background-image: var(--icon-circles-dark);
 }
 .icon-guests {
-	@include icon-color('app', 'guests', variables.$color-black);
+	background-image: var(--icon-guests-dark);
```
Comment (szaimen) [inline apps/files_sharing/css/icons.scss:33]: @nextcloud/frontenders it seems like this variable does not exist. How can I set or create it?

Comment (szaimen) [reply apps/files_sharing/css/icons.scss:33]: ping @skjnldsv @PVince81 ? :)

Comment (skjnldsv) [reply apps/files_sharing/css/icons.scss:33]:
```suggestion
	background-image: var(--icon-user-dark);
```

Comment (szaimen) [reply apps/files_sharing/css/icons.scss]: thanks! :)

**Review by skjnldsv — APPROVED** [review]

**Review by artonge — APPROVED** [review]

**Pull request merged**